docs/new-clients.txt

```diff
@@ -52,27 +52,78 @@ and commands with an object-oriented API in C++ and C.
 For more information, refer to the linkman:libnutclient[3] manual page.
 
   #include <iostream>
+  #include <unistd.h>
+  #include <stdlib.h>
+  
   #include <nutclient.h>
+  
   using namespace nut;
   using namespace std;
   
-  int main(int argc, char** argv)
+  int main(int argc, char **argv)
```

Comment:
@boricj nitpicking somehow, but:
a comment here in the code would be helpful to depict that
// argv[1] is the mandatory NUT device name (@localhost), used to list variables from
// argv[2] is an optional command. When provided, it will be executed and possibly with status tracking enabled
I'll merge the PR and do it myself to avoid another 1d delay